build.js

```diff
@@ -195,6 +196,32 @@ function buildlocale (source, locale) {
   })
 }
 
+// This middleware adds "Edit on GitHub" links to every editable page
+function githubLinks (options) {
+  return function (files, m, next) {
+    // add suffix (".html" or "/") to each part of regex
+    // to ignore possible occurencies in titles (e.g. blog posts)
+    var isEditable = /security\.html|about\/|docs\/|foundation\/|get\-involved\/|knowledge\//
+
+    Object.keys(files).forEach(function (path) {
+      if (!isEditable.test(path)) {
+        return
+      }
+
+      var file = files[path]
+      var url = 'https://github.com/nodejs/nodejs.org/edit/master/locale/' + options.locale + '/' + path.replace('.html', '.md')
```

Comment:
Does the markdown parser have a built-in api to do this? This is a bit hardcoding IMO.